plugins/modules/aci_interface_policy_link_level.py

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Vasily Prokopov (@vasilyprokopov)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'certified'}
+
+DOCUMENTATION = r'''
+---
+module: aci_interface_policy_link_level
+short_description: Manage Link Level interface policies (fabric:HIfPol)
+description:
+- The host interface policy specifies the layer 1 parameters of host facing ports.
```

Comment:
The link level interface policy specifies the layer 1 parameters of switch interfaces.